lib/views/home.dart

```diff
@@ -0,0 +1,296 @@
+                              RowAlbumCard(
+                                label: "Eminem",
+                                image: AssetImage("assets/album5.jpg"),
+                              ),
+                            ],
+                          ),
+                          const SizedBox(height: 16),
+                          Row(
+                            children: const [
+                              RowAlbumCard(
+                                label: "Top 50 - USA",
+                                image: AssetImage("assets/album9.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              RowAlbumCard(
+                                label: "Pop Remix",
+                                image: AssetImage("assets/album10.jpg"),
+                              ),
+                            ],
+                          ),
+                        ],
+                      ),
+                    ),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Text(
+                            "Based on your recent listening",
+                            style: Theme.of(context).textTheme.headline6,
+                          ),
+                        ),
+                        SingleChildScrollView(
+                          scrollDirection: Axis.horizontal,
+                          physics: const BouncingScrollPhysics(),
+                          padding: const EdgeInsets.symmetric(
+                            horizontal: 20,
+                          ),
+                          child: Row(
+                            children: const [
+                              SongCard(
+                                image: AssetImage("assets/album2.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              SongCard(
+                                image: AssetImage("assets/album6.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              SongCard(
+                                image: AssetImage("assets/album9.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              SongCard(
+                                image: AssetImage("assets/album4.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              SongCard(
+                                image: AssetImage("assets/album5.jpg"),
+                              ),
+                              SizedBox(width: 16),
+                              SongCard(
+                                image: AssetImage("assets/album1.jpg"),
```

Comment:
Same here, give a try to ListView builder or anything similar

Reply:
Same as above, create an array of song images:
```
const songs = [
  'assets/img1.png',
  'assets/img2.png',
  ...
]
```
and then just map each image to a `SongCard` widget